src/PortfolioPlanning/Common/Services/PortfolioPlanningDataService.ts

```diff
@@ -1058,4 +1035,260 @@ export class ODataQueryBuilder {
 
         return result;
     }
+
+    /**
+     * 
+    $select=WorkItemLinkSK,SourceWorkItemId,TargetWorkItemId,LinkTypeReferenceName,ProjectSK
```

Comment:
Will we be using this?

Reply:
yes, we are... I leave a comment on all these methods to remember what query I was trying to build :)